website/content/api-docs/health.mdx

```diff
@@ -235,7 +235,7 @@ The table below shows this endpoint's support for
 - `near` `(string: "")` - Specifies a node name to sort the node list in
   ascending order based on the estimated round trip time from that node. Passing
   `?near=_agent` uses the agent's node for the sort.
-  **Note** that using `near` will ignore 
+  **Note** that using `near` will ignore
   [`use_streaming_backend`](/docs/agent/config/config-files#use_streaming_backend) and always
   use blocking queries, because the data required to sort the results is not available
   to the streaming backend.
```

Comment:
```suggestion
  ~> **Note:** Using `near` will ignore
  [`use_streaming_backend`](/docs/agent/config/config-files#use_streaming_backend) and always
  use blocking queries, because the data required to sort the results is not available
  to the streaming backend.
```